.github/CONTRIBUTING.md

```diff
@@ -38,6 +38,54 @@ Tabs are boxes of content that show a list of tab titles, and the reader can vie
 
 To start a group of tabs, use the `{% tabs %}` tag. For each single tab, wrap them in `{% tab title="TITLE" %}`, replacing the title accordingly, and `{% endtab %}`. End the whole group of tabs with `{% endtabs %}`.
 
+## Style Guide
+
+### Do:
+
+- Communicate! Ask questions and make suggestions. It's ok to be wrong; Learn and try again, we love it!
+- Always add a hyperlink to the command's page when mentioned
+- Submit pull requests to the testing branch, not the main branch
+- Retain as much information as possible while remaining concise. It can be a difficult balance at times, but mistakes are ok! Pull requests and the testing branch allow for plenty of feedback and revision
+- Add additional relevant information when it is missing or improves the section
+- Use info hints for information that is indirectly related
+- Structure pages and sections in the order a user would need to set the bot up, then use comfortably, then in order of importance. I.e., make sure the bot works, then set it to a comfortable language, then continue to moderation set up
+- Tables containing specific values, such as the log names, should be alphabetical
+- Keep the main page as a summary/overview; Add the specific stuff to the dedicated/relevant page
+- Add explanations for suggestions. Your way of thinking might not be immediately clear to others, but that doesn't make it bad or wrong; Even if it's just a gut feeling, let us know!
+
+
+Screenshots:
+
+- Add screenshots where relevant, clarifying, needed, etc.
+- Set your display name to `MoonlightCapital`
+- Set your profile picture to [this image](https://cdn.discordapp.com/attachments/484218269662707734/1362150383258173491/davatar3.png)
```

Comment:
This doesn't really work, CDN links sent outside of Discord expire after a while.
On these points, add the reason why I'm being impersonated (with my consent)